src/applications/vaos/utils/data.js

```diff
@@ -44,6 +45,29 @@ function getUserMessage(data) {
   return `${label}: ${data.reasonAdditionalInfo}`;
 }
 
+// When useVSP is true, the clinic data is using the FHIR format. The clinic id format is:
+// urn:va:healthcareservice:983:983:308
+// where the last 3 tokens represent the site id, facility id, and clinic id.
+function getToken(clinic, index) {
```

Comment:
`getToken` is a pretty generic name, what about something like `getURNToken` or `getIdentifierToken`?

Reply:
Done.